packages/matter-node.js/README_EXAMPLES.md

```diff
@@ -56,6 +56,7 @@ The following parameters are available:
 * -discriminator: the discriminator to use for pairing (default: 3840, value between 0 and 4095)
 * -vendorid: the vendor ID as number to use for pairing (default: 65521 (0xFFF1))
 * -productid: the product ID as number to use for pairing (default: 32768 (0x8000))
+* -uniqueid: a unique ID for this device to be used in naming and to stire structure information (default: ms time now)
```

Comment:
I'm not familiar with "stire", is this a typo for "store"?

Reply:
fixed